supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -56,8 +56,18 @@ object JesAsyncBackendJobExecutionActor {
   val JesUnexpectedTermination = 13
   val JesPreemption = 14
 
-  def StandardException(errorCode: Status, message: String, jobTag: String) = {
-    new Exception(s"Task $jobTag failed. JES error code ${errorCode.getCode.value}. $message")
+  def StandardException(errorCode: Status,
+                        message: String,
+                        jobTag: String,
+                        returnCodeOption: Option[Int],
+                        stderrPath: Path) = {
+    val returnCodeMessage = returnCodeOption match {
+      case Some(returnCode) if returnCode == 0 => " Job exited without an error, exit code 0."
+      case Some(returnCode) => s" Job exit code $returnCode. Check $stderrPath for more information."
+      case None => ""
```

Comment:
Actually I'd say that this could be useful information (`"the job was stopped before the command finished"`). In fact I'd guess this scenario could be the majority of exceptional cases.

Reply:
Side benefit of adding text here would be that we can move the space (` `) to the containing string. Woohoo!